src/runner/settings_window.cpp

```diff
@@ -151,34 +151,30 @@ void dispatch_received_json(const std::wstring& json_to_parse)
     const json::JsonObject j = json::JsonObject::Parse(json_to_parse);
     for (const auto& base_element : j)
     {
+        if (!current_settings_ipc)
+        {
+            continue;
```

Comment:
Can this cause the Settings page to not render all the data?

Reply:
well, we could have an empty `Last checked:` in some cases

Reply:
the proper fix would be to remove the IPC race condition of course